.github/workflows/ci.yml

```
@@ -45,3 +45,15 @@ jobs:
 
       - name: Verify that canonical-data.json files adhere to canonical-schema.json
         run: yarn test
+      
+      - name: Verify that all UUIDs are indeed unique
+        run: |
+          duplicate_uuids=$(cat exercises/*/canonical-data.json |
+              grep '"uuid":' |                                    # select UUID line
+              sed -E 's/.*"([^"]+)".*/\1/' |                      # extract JSON value
```

```suggestion
              jq -r '.. |."uuid"? | select(. != null)' |          # select all UUIDs
```
Just in case we wanted some `jq` voodoo.

For me, the `sed` is more familiar, but the `jq` version seems nice.